src/main/java/com/derongan/minecraft/mineinabyss/world/LayerImpl.kt

```diff
@@ -4,7 +4,9 @@ import com.derongan.minecraft.deeperworld.world.section.Section
 import com.derongan.minecraft.mineinabyss.ascension.effect.AscensionEffectBuilder
 import java.util.*
 
-class LayerImpl(override val name: String, override val sub: String, override val index: Int, override val deathMessage: String?) : Layer {
+class LayerImpl(override val name: String, override val sub: String, override val index: Int, override val deathMessage: String?,
+                override val maxCurseRadius: Float, override val minCurseRadius: Float, override val minCurseMultiplier: Float,
```

Comment:
For a future PR maybe we should consider putting all these curse related attributes in an object. Right now our constructor is getting a tad bit ugly.

Reply:
yes open task #37